extensions.json

```diff
@@ -261,18 +261,18 @@
             "version": 0.1,
             "entrypoint": "TTRSS_API",
             "type": "system",
-            "url": "https:\/\/github.com\/FreshRSS\/Extensions",
+            "url": "https://github.com/FreshRSS/Extensions",
             "method": "git",
             "directory": "xExtension-TTRSS_API"
         },
         {
             "name": "Title-Wrap",
-            "author": "\u20a3rans de Jonge, Joris Kinable",
+            "author": "₣rans de Jonge, Joris Kinable",
```

Comment:
Is the ₣ on purpose, @Frenzie ? :-)

Reply:
No, see https://github.com/FreshRSS/Extensions/pull/105

Reply:
It is because it is not fixed in https://github.com/FreshRSS/Extensions/blob/master/xExtension-TitleWrap/metadata.json